### ISSUE TITLE
Crash on creating a new model

### ISSUE THREAD

**Issue by nathanbjenx** [failys/cairis#46]
### Expected behaviour
New empty model created.

### Actual behaviour
Program hangs, then once the sql query is killed produces: 
`sqlalchemy.exc.ProgrammingError

ProgrammingError: (_mysql_exceptions.ProgrammingError) (1146, "Table 'cairis.roles_users' doesn't exist") [SQL: u'SELECT auth_role.id AS auth_role_id, auth_role.name AS auth_role_name, auth_role.description AS auth_role_description \nFROM auth_role, roles_users \nWHERE %s = roles_users.user_id AND auth_role.id = roles_users.role_id'] [parameters: (1L,)]
`

### Steps to reproduce the behaviour

1. Log in to CAIRIS
2. Select new from system menu

### Version of CAIRIS

All versions

### Operating Systems / Hardware 

All except when using docker

**Comment by failys** [comment]
I can't reproduce the problem, but the fact that the tables used for authentication are removed when the database is cleared might not be helping!  

Can you please pull the latest sources and try creating a new project using the API only, e.g. from a shell session, run:
    curl -X POST http://localhost:7071/api/settings/create?session_id=test

**Comment by nathanbjenx** [comment]
I worked around this problem yesterday by changing the configure_db script to put the authentication tables in a separate database. However after pulling the latest sources the bug disappeared!

After racking my brain for what I'd done differently before I realised that for whatever reason I'd decided to install it with the setup.py script.

So in short this is a bug that doesn't actually exist in the current version!
